content/en/service_management/service_level_objectives/monitor.md

```diff
@@ -112,7 +112,19 @@ Muting a monitor does not affect the SLO calculation.
 To exclude time periods from an SLO calculation, use the [SLO status corrections][5] feature.
 
 ### Missing data
-When you create a metric monitor or service check, you choose whether it sends an alert when data is missing. This configuration affects how a monitor-based SLO calculation interprets missing data. For monitors configured to ignore missing data, time periods with missing data are treated as OK (uptime) by the SLO. For monitors configured to alert on missing data, time periods with missing data are treated as ALERT (downtime) by the SLO.
+#### Metric monitors
+When you create a metric monitor, you choose [how the monitor will handle missing data][6]. This configuration affects how a monitor-based SLO calculation interprets missing data:
+
+| Monitor configuration     | SLO calculation of missing data |
+|---------------------------|---------------------------------|
+| `Evaluate as zero`        | ???                             |
```

Comment:
Up to underlying monitor status set by comparing zero with alert threshold.  For example, if the alert threshold is set to > 10,  the monitor status would be set to OK. Thus,  Uptime in SLO calculation